config.json

```diff
@@ -11,404 +11,404 @@
       ]
     },
     {
-      "slug": "accumulate",
+      "slug": "space-age",
       "difficulty": 1,
       "topics": [
       ]
     },
     {
-      "slug": "sublist",
+      "slug": "bob",
       "difficulty": 1,
       "topics": [
       ]
     },
     {
-      "slug": "grains",
+      "slug": "rna-transcription",
       "difficulty": 1,
       "topics": [
       ]
     },
     {
-      "slug": "bob",
+      "slug": "sum-of-multiples",
       "difficulty": 1,
       "topics": [
       ]
     },
     {
-      "slug": "sum-of-multiples",
+      "slug": "difference-of-squares",
       "difficulty": 1,
       "topics": [
       ]
     },
     {
-      "slug": "strain",
-      "difficulty": 1,
+      "slug": "grains",
+      "difficulty": 2,
       "topics": [
       ]
     },
     {
       "slug": "hamming",
-      "difficulty": 1,
+      "difficulty": 2,
       "topics": [
       ]
     },
     {
-      "slug": "rna-transcription",
-      "difficulty": 1,
+      "slug": "nucleotide-count",
+      "difficulty": 2,
       "topics": [
       ]
     },
     {
-      "slug": "space-age",
-      "difficulty": 1,
+      "slug": "grade-school",
+      "difficulty": 2,
       "topics": [
       ]
     },
     {
-      "slug": "anagram",
-      "difficulty": 1,
+      "slug": "etl",
+      "difficulty": 2,
       "topics": [
       ]
     },
     {
-      "slug": "phone-number",
-      "difficulty": 1,
+      "slug": "raindrops",
+      "difficulty": 2,
       "topics": [
       ]
     },
     {
-      "slug": "nucleotide-count",
-      "difficulty": 1,
+      "slug": "accumulate",
+      "difficulty": 3,
       "topics": [
       ]
     },
     {
-      "slug": "grade-school",
-      "difficulty": 1,
+      "slug": "strain",
+      "difficulty": 3,
       "topics": [
       ]
     },
     {
-      "slug": "simple-linked-list",
-      "difficulty": 1,
+      "slug": "phone-number",
+      "difficulty": 3,
       "topics": [
       ]
     },
     {
-      "slug": "list-ops",
-      "difficulty": 1,
+      "slug": "beer-song",
```

Comment:
really? Is beer song really easier than house? Of the three, I thought the difficulty is house < beer song < food chain.